src/other/utils.jl

```diff
@@ -49,14 +49,21 @@ function makeidentifier(s::AbstractString)
     return takebuf_string(res)
 end
 
-function make_unique(names::Vector{Symbol})
+function make_unique(names::Vector{Symbol}; allow_duplicates=true)
     seen = Set{Symbol}()
     names = copy(names)
     dups = Int[]
     for i in 1:length(names)
         name = names[i]
         in(name, seen) ? push!(dups, i) : push!(seen, name)
     end
+    
+    if (!allow_duplicates) && length(dups) > 0
+        d = unique(names[dups])
+        msg = "Duplicate variable names: $d. Set allow_duplicates to true to fix it."
```

Comment:
Maybe even more explicit: `Pass allow_duplicates=true to make them unique using a suffix automatically.`